docs/authentication/configuration/sign-up-options.mdx

```diff
@@ -48,7 +48,7 @@ For each provider, Clerk offers a shared profile that can be used for developmen
 
 ## Choose Web3 authentication
 
-Clerk provides [Web3 authentication](/docs/users/web3) as an option with MetaMask. As part of validating the accuracy of the returned Web3 account address, Clerk handles the signing of a transaction and verifying the signature. Because sign-in with MetaMask uses the same abstraction as our other authentication strategies, like passwords or magic links, other Clerk features like multi-factor authentication and profile enrichment work for MetaMask users out-of-the-box.
+Clerk provides [Web3 authentication](/docs/users/web3) as an option with MetaMask. As part of validating the accuracy of the returned Web3 account address, Clerk handles the signing of a transaction and verifying the signature. Because sign-in with MetaMask uses the same abstraction as our other authentication factors, like passwords or email links, other Clerk features like multi-factor authentication and profile enrichment work for MetaMask users out-of-the-box.
```

Comment:
```suggestion
Clerk provides [Web3 authentication](/docs/users/web3) as an option with MetaMask. As part of validating the accuracy of the returned Web3 account address, Clerk handles the signing of a transaction and verifying the signature. Because sign-in with MetaMask uses the same abstraction as our other authentication strategies, like passwords or email links, other Clerk features like multi-factor authentication and profile enrichment work for MetaMask users out-of-the-box.
```